portfolio/src/main/webapp/index.html

```diff
@@ -74,10 +74,13 @@ <h1>Life Before Quarantine</h1>
         </div>
       </div>
     </section>
-    <section id="comments">
-      <button type="button" class="btn btn-outline-primary" 
-      onclick="getComments()">See Comments</button>
+    <section id="comments" >
       <ul id="comments-grid"></ul>
+      <form action="/data" method="POST">
+        <label for="comment-box">Drop a Comment!</label><br>
```

Comment:
I'd consider leaving out the `br` tags and using CSS to adjust styles instead. My overarching goal is not to add things to HTML that box me into a specific visual appearance, and the same goes with break. By doing this, you're saying that you always want each of those on their own line; but what if you go to redesign your site and have a great new layout that keeps them all on one line? Really nice to be able to stay in your CSS to adjust styles and not need to update your HTML. Also it just makes the HTML cleaner. :)